bin/setup.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Check if .env file already exists
+if [ -f .env ]; then
+  echo ".env file already exists. Please remove it first."
```

Comment:
We can never know for sure why a user might already have a .env file, we maybe dont want to assume it needs to be removed. Let's also be more generic in our wording here; `".env.example file does not exist. Aborting."`